tests/parser/ParserTests.cpp

```diff
@@ -1856,3 +1856,31 @@ UDQ
     BOOST_CHECK_EQUAL_COLLECTIONS( data0.begin(), data0.end(), expected0.begin(), expected0.end());
     BOOST_CHECK_EQUAL_COLLECTIONS( data1.begin(), data1.end(), expected1.begin(), expected1.end());
  }
+
+
+BOOST_AUTO_TEST_CASE(ParseThreePhaseRelpermModels) {
```

Comment:
I would have merged this without tests - but anyway; no harm in having them.

Reply:
> I would have merged this without tests

Shock!  Horror!

> - but anyway; no harm in having them.

To be honest, they don't really test anything other than the fact that the parser now recognises those keywords.